terminal.go

```diff
@@ -818,16 +818,19 @@ func (g *LightningTerminal) setUpLNDClients(lndQuit chan struct{}) error {
 	for {
 		g.lndClient, err = lndclient.NewLndServices(
 			&lndclient.LndServicesConfig{
-				LndAddress:            host,
-				Network:               network,
-				TLSPath:               tlsPath,
-				Insecure:              insecure,
-				CustomMacaroonPath:    macPath,
-				CustomMacaroonHex:     hex.EncodeToString(macData),
+				LndAddress:         host,
+				Network:            network,
+				TLSPath:            tlsPath,
+				Insecure:           insecure,
+				CustomMacaroonPath: macPath,
+				CustomMacaroonHex: hex.EncodeToString(
+					macData,
+				),
 				BlockUntilChainSynced: true,
 				BlockUntilUnlocked:    true,
 				CallerCtx:             ctxc,
 				CheckVersion:          minimalCompatibleVersion,
+				ChainSyncPollInterval: g.cfg.LndConnectInterval,
```

Comment:
nice

Reply:
Yeah, should speed up the normal itest as well.